com.Cisco.PacketTracer.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.Cisco.PacketTracer
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: packettracer
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --persist=pt # App config, logs, templates, saves stored in: ~/pt
+
+modules:
+  - name: packettracer
+    buildsystem: simple
+    build-commands:
+      - mkdir packettracer
+      - ar -x CiscoPacketTracer_821_Ubuntu_64bit.deb --output=packettracer
+      - tar -xvf packettracer/control.tar.xz --directory=packettracer
+      - tar -xvf packettracer/data.tar.xz --directory=packettracer
+      - chmod -R 755 packettracer
+      - mv packettracer/usr/share packettracer/share
+      - rm -d packettracer/usr
+      - cp -rf packettracer/. /app/
+      - sed -i 's/\/opt\/pt/\/app\/opt\/pt/g' /app/opt/pt/packettracer
+      - install -D /app/opt/pt/packettracer /app/bin/packettracer
+      - install -D /app/opt/pt/art/app.png /app/share/icons/hicolor/48x48/apps/$FLATPAK_ID.png
```

Comment:
minimum acceptable icon size is 128x128.

Reply:
The application only makes available a 48x48 official icon, there is no 128x128 official packet tracer icon. What should I do in this case?

Reply:
put a sha256.